meltano.yml

```diff
@@ -8,23 +8,30 @@ plugins:
   extractors:
   - name: tap-file
     namespace: tap_file
-    pip_url: -e .[s3]
+    pip_url: -e .
```

Comment:
we don't want to install for s3 by default anymore?

Reply:
Is that what we ended up deciding? I don't remember. When I saw that it was installed by default I thought it was an error that I'd forgotten to remove.